docker/db/mysql/init/create_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE GAMES(
+    game_id int NOT NULL,
```

Comment:
```suggestion
    id int NOT NULL,
```
이미 게임이라는 테이블이기 때문에 id라고 명명하는게 적절할것 같습니다!

Reply:
id의 경우 BIGINT를 사용하는게 어떨까요?

Reply:
식별자는 조금 널널하게 값의 범위를 제공해 주는 것이 좋겠네요!